tools/ci/sql_v0.sql

```diff
@@ -535,3 +535,4 @@ INSERT INTO `whitelist` (`id`, `ckey`, `job`, `species`) VALUES
 /*!40101 SET SQL_MODE=IFNULL(@OLD_SQL_MODE, '') */;
 /*!40014 SET FOREIGN_KEY_CHECKS=IF(@OLD_FOREIGN_KEY_CHECKS IS NULL, 1, @OLD_FOREIGN_KEY_CHECKS) */;
 /*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
+SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));
```

Comment:
This is required to stop CI MariaDB from complaining about empty selects when trying to order by columns. In production, this will never happen, and I couldnt even repro it locally with a fresh database. Its just CI being such a blank slate.